runner.py

```diff
@@ -296,6 +296,29 @@ def validate_input(output_dir, input_dir, reference_file, mode):
         raise Exception("Could not find files ending with '.fastq' or 'fastq.gz' in input_dir !")
 
 
+def create_stats_file(output_dir, filenames, overlapping_reads, reads_input_num):
+    file_path = os.path.join(output_dir, "stats_file.stats")
+    called_bases_path = os.path.join(output_dir, "called_bases.tsv")
+    dict_states = {'N': '1', 'M': '1 or 2', 'Y': 'exactly 2'}
+    with open(filenames['read_counter_file'], 'r') as read_counter:
+        counter_pd = pd.read_csv(read_counter, sep="\t", engine='python')
+        len_counter = str(len(counter_pd.read_id))
```

Comment:
len(counter_pd) would give the same result

Reply:
changed